### PR TITLE
UI polish

### PR DESCRIPTION
**Pull request by kevinkace**

### CODE REVIEW

File: src/pages/content-edit/head.css [hunk 1]
```diff
@@ -1,36 +1,22 @@
 @value pure: "../../_pure.css";
-@value   icons_css: "../../icons.css";
-@value  layout_css: "../layout/layout.css";
+@value icons_css: "../../icons.css";
+@value layout_css: "../layout/layout.css";
 @value buttons_css: "../../buttons.css";
 @value header: "../layout/header.css";
 
-.head {
+.contentHead {
```
Comment (kevinkace) [inline src/pages/content-edit/head.css]: Code imperialism.

Changed back to `.contentHead`. What do you think about renaming the content body to `.contentBody`? Getting a grasp of the different sections was a little tricky at first.

```
+--------------------------------------------------+
|                   container                      |
| +----------+ +---------------------------------+ |
| |  header  | |             content             | |
| | +------+ | | +-----------------------------+ | |
| | | head | | | |         contentHead         | | |
| | +------+ | | +-----------------------------+ | |
| | +------+ | | +-----------------------------+ | |
| | | body | | | |             body            | | |
| | |      | | | |                             | | |
| | |      | | | |                             | | |
| | |      | | | |                             | | |
| | |      | | | |                             | | |
| | +------+ | | +-----------------------------+ | |
| |          | |                                 | |
| +----------+ +---------------------------------+ |
|                                                  |
+--------------------------------------------------+
```

Comment (tivac) [inline src/pages/content-edit/head.css]: I like it, prefer `.contentHd` and `.contentBd` though.

Naming things is a PITA.

Comment (kevinkace) [inline src/pages/content-edit/head.css]: Ya, seriously.

Comment (tivac) [inline src/pages/content-edit/head.css]: :+1:

**Comment by tivac** [comment]
@kevinkace 😕❓

**Comment by kevinkace** [comment]
Ya, sorry, in merge hell. I'll hopefully get it sorted tonight.

**Comment by tivac** [comment]
Ok, let me know if I can help!

**Comment by kevinkace** [comment]
Lol, didn't realize *I* closed this PR. Probably for the best though. New PR this afternoon!

**Pull request closed without merging**